castor-service/src/test/java/io/carbynestack/castor/service/download/CreateReservationSupplierTest.java

```diff
@@ -111,21 +113,53 @@ public void givenSharingReservationFails_whenGet_thenThrowCastorServiceException
   }
 
   @Test
-  public void givenSuccessfulRequest_whenGet_thenReturnExpectedReservation() {
-    TupleChunkMetaDataEntity metaDataEntityMock = mock(TupleChunkMetaDataEntity.class);
-    UUID chunkId = UUID.fromString("c8a0a467-16b0-4f03-b7d7-07cbe1b0e7e8");
+  public void givenMultipleChunksWithAvailable_whenGet_thenReturnExpectedReservation() {
+    TupleChunkMetaDataEntity metaDataEntity1Mock = mock(TupleChunkMetaDataEntity.class);
+    TupleChunkMetaDataEntity metaDataEntity2Mock = mock(TupleChunkMetaDataEntity.class);
+    UUID chunk1Id = UUID.fromString("c8a0a467-16b0-4f03-b7d7-07cbe1b0e7e8");
+    UUID chunk2Id = UUID.fromString("4d206204-96a0-4ddc-b409-4ac05b1f12f2");
```

Comment:
Codacy complains that this variable is never used, which seems to be true.

Reply:
has been addressed in commit 721abbe. 
Hoewever, Test naming convention will not be changed for now